tests/model_registry/test_model_registry_creation.py

```diff
@@ -32,28 +31,51 @@ class TestModelRegistryCreation:
     # TODO: Switch to Python client
     @pytest.mark.smoke
     def test_registering_model(
-        self: Self,
-        model_registry_instance_rest_endpoint: str,
-        current_client_token: str,
-        updated_dsc_component_state,
+        self: Self, model_registry_instance_rest_endpoint: str, current_client_token: str, updated_dsc_component_state
     ):
-        cmd = generate_register_model_command(
-            endpoint=model_registry_instance_rest_endpoint, token=current_client_token
+        # address and port need to be split in the client instantiation
+        server = model_registry_instance_rest_endpoint.split(":")[0]
+        port = model_registry_instance_rest_endpoint.split(":")[1]
+        registry = ModelRegistry(
+            server_address=f"{Protocols.HTTPS}://{server}",
+            port=port,
+            author="opendatahub-test",
+            user_token=current_client_token,
+            is_secure=False,
+        )
+        model = registry.register_model(
+            name="my-model",  # model name
+            uri="https://storage-place.my-company.com",  # model URI
+            version="2.0.0",
+            description="lorem ipsum",
+            model_format_name="onnx",
+            model_format_version="1",
+            storage_key="my-data-connection",
+            storage_path="path/to/model",
+            metadata={
+                # can be one of the following types
```

Comment:
the comment to me is a bit confusing - what does it mean `can be one of the following types`? in which usage?

Reply:
When adding metadata to a model you're registering in MR, just there for easy accessibility while developing in the same file new tests

Reply:
Removed a few comments from the latest version, resolving this thread